admin-front/src/shared/hooks/useGetPriceWithCurrency.tsx

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useState } from 'react';
+import {
+  Currency,
+  CurrencySymbol,
+  CurrencySymbolData,
+  SupportedCurrency,
+} from '../types/currency';
+import { useNotify } from 'react-admin';
+import axios from 'axios';
+
+interface GetBaseCurrencyFromAPI {
+  notify: (value: string) => void;
+}
+
+const getBaseCurrencyFromAPI = async ({
+  notify,
+}: GetBaseCurrencyFromAPI): Promise<SupportedCurrency> => {
+  try {
+    const response = await axios.get(
+      process.env.REACT_APP_STORE_BASE_URL + 'api/constants',
+    );
+
+    const { baseCurrency } = response.data;
+
+    return baseCurrency;
+  } catch (e) {
+    notify('An error occurred while fetching the base currencies');
+  }
+
+  return '';
+};
+
+const getCurrencySymbolDataForCurrency = (
+  currency: SupportedCurrency,
+): CurrencySymbolData | undefined => {
+  switch (currency) {
+    case Currency.EUR:
+      return { symbol: CurrencySymbol.EUR, position: 'after' };
+    case Currency.XTZ:
+      return { symbol: CurrencySymbol.XTZ, position: 'before' };
```

Comment:
Tez symbol should go after the value. For example see tzkt, a popular block explorer in tezos: https://tzkt.io/

Reply:
Thanks for clarifying. I interpreted it this way because of Wikipedia.

Reply:
Aha, interesting. Then I guess we can put it wherever we want to. Not sure which is correct either then.